docs/bibliography.rst

```diff
@@ -0,0 +1,5 @@
+Bibliography
+============
+
```

Comment:
I might add something like "the following were referenced during the development of stardis" to note that this is the bibliography for the code as a whole, not just for the documentation (as is the case with TARDIS).

Reply:
[Does this look good?](https://github.com/tardis-sn/stardis/pull/81/commits/d1ee1b9255eab1d505556dc25feea035afdd7cdd)